kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/ZooKeeperHiveClientHelper.java

```diff
@@ -111,6 +112,7 @@ static void configureConnParams(JdbcConnectionParams connParams)
       String serverNode = serverHosts.get(new Random().nextInt(serverHosts.size()));
       updateParamsWithZKServerNode(connParams, zooKeeperClient, serverNode);
     } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
we don't need it, the existing code is sufficient to propagate underlying exception stacktrace to user.

Reply:
Thanks for your suggestion.

On the basis of the current code, I tried several methods and failed to output the exception stacktrace.

I refer to the following: https://github.com/apache/kyuubi/blob/master/kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiBaseResultSet.java#L300

So what is the specific way you say?

Reply:
Actually we should remove the `e.printStackTrace();` it's totally a bad practice.

> ...  I tried several methods and failed to output the exception stacktrace.

The caller is responsible to catch and handle the exception. i.e. the caller may want to catch and print the stacktrace using log system. 
```
Logger log = LoggerFactory.getLogger(xxx);
try {
  ...
} catch (Exception cause) {
  log.error(cause.getMessage(), cause)
}
```